client/src/components/Sidebar.jsx

```diff
@@ -71,6 +71,12 @@ export default function TemporaryDrawer() {
 							<ListItemText primary="Home" />
 						</ListItemButton>
 					</ListItem>
+					<ListItem key="About Us" disablePadding>
+						<ListItemButton onClick={() => navigate("/aboutus")}>
+							<ListItemIcon>{getIconByLabel("AboutUs")}</ListItemIcon>
+							<ListItemText primary="About us" />
+						</ListItemButton>
+					</ListItem>
```

Comment:
in line with the name changes above

```suggestion
					<ListItem key="About" disablePadding>
						<ListItemButton onClick={() => navigate("/about")}>
							<ListItemIcon>{getIconByLabel("About")}</ListItemIcon>
							<ListItemText primary="About" />
						</ListItemButton>
					</ListItem>
```

Reply:
<img width="1440" alt="Screenshot 2023-11-14 at 19 34 33" src="https://github.com/fazbazjaz/star/assets/113185323/f2fd27c0-271f-411a-b52f-e54478c10d22">